src/datapump/jobs/geotrellis.py

```diff
@@ -834,6 +833,15 @@ def _run_job_flow(self, name, instances, steps, applications, configurations):
         if GLOBALS.emr_service_role:
             request["ServiceRole"] = GLOBALS.emr_service_role
 
+        # If using version 2.4.1 or earlier, use older GDAL version
+        if self.geotrellis_version < "2.4.1":
```

Comment:
Not sure what the type of self.geotrellis_version is, but you're comparing it to a string here. Is that going to work?

Reply:
I think it is a string